### PR TITLE
115 nyaml2nxdl xref edge cases

### PR DESCRIPTION
**Pull request by RubelMozumder**
changes:
1. `term` will accept anything in-between a `space` and `standard`.
2. Any kind of URLs will be accepted whether they are valid or not.

### CODE REVIEW

**Comment by prjemian** [comment]
- This PR does not reference an issue.  It should describe *why* this branch and PR was created.  (The opening statement is limited in this regard.)
- Why are all the existing NeXus class definitions added here as nyaml files?

File: dev_tools/nyaml2nxdl/nyaml2nxdl_forward_tools.py [hunk 1]
```diff
@@ -285,7 +285,7 @@ def handle_each_part_doc(text):
     Formated text
     """
 
-    clean_txt = text.strip()
+    clean_txt = text.strip().strip('"')
```
Comment (prjemian) [inline dev_tools/nyaml2nxdl/nyaml2nxdl_forward_tools.py:288]: Would removal of starting `"` use `lstrip('"')` instead?

**Comment by domna** [comment]
> * This PR does not reference an issue.  It should describe _why_ this branch and PR was created.  (The opening statement is limited in this regard.)
> * Why are all the existing NeXus class definitions added here as nyaml files?

Sorry, I don't know why this PR is open here. This is a current PR for the fairmat nexus repository (I only saw accidentally that it is opened here as well). This should only be contributed with #1303 or at a later stage.

@RubelMozumder Was this PR accidentally created and if so can you close it here?

File: base_classes/NXroot.nxdl.xml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
 <!--
 # NeXus - Neutron and X-ray Common Data Format
-#
-# Copyright (C) 2008-2022 NeXus International Advisory Committee (NIAC)
-#
+# 
+# Copyright (C) 2014-2023 NeXus International Advisory Committee (NIAC)
```
Comment (prjemian) [inline base_classes/NXroot.nxdl.xml:6]: As stated [elsewhere](https://github.com/nexusformat/definitions/pull/1303#issuecomment-1806853868), do not change the copyright dates

**Comment by RubelMozumder** [comment]
It was created by mistake.

**Pull request closed without merging**